tools/cli/README.md

```diff
@@ -158,10 +158,25 @@ There are a lot of use cases:
 - Rerun a failed workflow from the failing point without losing the achieved progress(history).
 - After deploying new code, reset an open workflow to let the workflow run to different flows.
 
+You can reset to some predefined event types:
+```
+./cadence workflow reset -w <wid> -r <rid> --reset_type <reset_type> --reason "some_reason"
+```
+
+- FirstDecisionCompleted: reset to the beginning of the history.
+- LastDecisionCompleted: reset to the end of the history.
+- LastContinuedAsNew: reset to previous run's end of the history
```

Comment:
reset to previous run's end of the history -> reset to the end of the history for the previous run.